lib/chai/utils/inspect.js

```diff
@@ -318,6 +352,12 @@ function reduceToSingleString(output, base, braces) {
   return braces[0] + base + ' ' + output.join(', ') + ' ' + braces[1];
 }
 
+function isTypedArray(ar) {
+  // Unfortunately there's no way to check if an object is a TypedArray
+  // We have to check if it's one of these types
```

Comment:
Another minor note:
I remember that when I first did this there was a list of types below this comment, now, since that list was removed it makes no sense anymore. Maybe we should find a way to update this comment line or even remove it. However, I'd like to change it to:

```js
// This regex checks if the object is one of the following types:
// Int8Array
// Int16Array
// Etc, etc, etc...
```